### PR TITLE
Use buffer instead of atob/btoa

### PR DESCRIPTION
**Pull request by Nio-o**
## Use buffer instead of atob/btoa to support utf8

Details described in issue https://github.com/webdriverio/webdriverio/issues/7710

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/webdriver/src/request/browser.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default class BrowserRequest extends WebDriverRequest {
         }
 
         if (options.username && options.password) {
-            const encodedAuth = btoa(`${options.username}:${options.password}`)
```
Comment (Nio-o) [inline packages/webdriver/src/request/browser.ts:41]: I'm not sure is it executed inside Browser or Node environment?

File: packages/wdio-devtools-service/src/gatherer/coverage.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import fs from 'fs'
 import path from 'path'
-import atob from 'atob'
```
Comment (Nio-o) [inline packages/wdio-devtools-service/src/gatherer/coverage.ts:3]: Actually I search across codebase and didn't found any usage of atob, but maybe I missing something. That's why I didnt remove it from package.json

File: packages/webdriver/src/request/browser.ts (hunk 1 above)
Comment (christian-bromann) [reply packages/webdriver/src/request/browser.ts:41]: always inside Node environment

File: packages/wdio-devtools-service/src/gatherer/coverage.ts (hunk 2 above)
Comment (christian-bromann) [reply packages/wdio-devtools-service/src/gatherer/coverage.ts:3]: we can remove it then from the package.json

Comment (Nio-o) [reply packages/wdio-devtools-service/src/gatherer/coverage.ts:3]: I removed atob everywhere except one place: [/webiste/package-lock.json](https://github.com/webdriverio/webdriverio/blob/main/website/package-lock.json). I don't why there is atob presented in lock file but missing in package.json. I didn't work with monorepositories and lerna, my be I don't why it works like that

**Comment by Nio-o** [comment]
Test / e2e-mac failed. Is it problem with pull request or not?

**Review by christian-bromann — APPROVED** [review]
> Test / e2e-mac failed. Is it problem with pull request or not?

No, these are usually flaky, I will continue and merge

Thanks a lot! 👍

**Comment by Nio-o** [comment]
@christian-bromann Hello. Can you tell when this fix will be released?

**Pull request merged**